src/main/java/edu/umich/its/cpm/BoxUtils.java

```diff
@@ -50,10 +68,55 @@
 
 	private static final String CODE = "code";
 
+	private static String box_access_token = null;
+
+	private static String box_refresh_token = null;
+	
+	private static final String BOX_CLIENT_ID = "box_client_id";
+	private static final String BOX_CLIENT_SECRET = "box_client_secret";
+	
+	/**
+	 * get Box access token
+	 */
+	public static String getBoxAccessToken()
+	{
+		return box_access_token;
+	}
+	
+	/**
+	 * set Box access token
+	 */
+	public static void setBoxAccessToken(String boxAccessToken)
+	{
+		box_access_token = boxAccessToken;
+	}
+	
+	
+	/**
+	 * get Box refresh token
+	 */
+	public static String getBoxRefreshToken()
+	{
+		return box_refresh_token;
+	}
+	
+	/**
+	 * set Box refresh token
+	 */
+	public static void setBoxRefreshToken(String boxRefreshToken)
+	{
+		box_refresh_token = boxRefreshToken;
+	}
+	
 	private static final SimpleDateFormat date_formatter = new SimpleDateFormat(
```

Comment:
Wrapped the use of SimpleDateFormat object with synchronized:

synchronized(date_formatter){
				rv = date_formatter.format(date);
			}